### DISCUSSION TITLE
Typo on the slide of the FL concept number 8

### DISCUSSION THREAD

**Discussion by gonzalo-munillag** [OpenMined/courses#271] [Feedback] [answered]
Privacy of sensitive data is (Stored) **preserved**

**Comment by yemikifouly** [comment, marked as answer]
Hello Gonzalo! 👋 I am really glad to see you back for course 2. How are you liking the Federated Learning lesson so far? 🙂 You are right. The slide should say "preserved".  Thank you for pointing this out. I will bring this to the content team. I am not completely sure that they will be able to fix the slide itself though but I will bring it to them nonetheless. 🙂 

Thanks again Gonzalo! And enjoy the lesson!

**Reply by gonzalo-munillag** [reply]
Hi Yemi! Great to be back and great to see you here again too hahah :)
I cannot tell yet how good the course will be, but I am very much looking forward to keep progressing, as I am planning to combine all these learnings with my research to write a research paper, starting in the next months. 
Looking forward to the next lessons!
Cheers!
